### DISCUSSION TITLE
Error: Cannot find module '../lightningcss.win32-x64-msvc.node'

### DISCUSSION THREAD

**Discussion by babakfp** [denoland/deno#23255] [Q&A]
Hi 👋

https://github.com/parcel-bundler/lightningcss/discussions/712

Based on this comment https://github.com/parcel-bundler/lightningcss/discussions/712#discussioncomment-9029656 and this:

![Screenshot 2024-04-06 223807](https://github.com/denoland/deno/assets/44144724/d10c2985-f70a-489e-9ffd-11f9157b20e3)

(I used this library https://deno.land/x/abc@v1.3.3 and when I compiled, it started to download this ^ file).

It seems like Deno doesn't do the same for lightningcss.

**Comment by marvinhagemeister** [comment]
Converted this into an issue (https://github.com/denoland/deno/issues/23266) because it's a bug in Deno where something is not working, rather than a discussion where users ponder about "what is the best way to do X" for example.

Issue: https://github.com/denoland/deno/issues/23266
